nncf/tensorflow/graph/transformations/commands.py

```diff
@@ -375,6 +375,18 @@ def from_state(cls, state: Dict[str, Any]) -> 'TFOperationWithWeights':
         return cls(**state)
 
 
+class CallableObject:
+    """
+    Stores the callable object and instance index
+    """
+    def __init__(self, callable_obj: Optional[Callable] = None, instance_idx: int = 0):
+        self.callable = callable_obj
+        self.instance_idx = instance_idx
+
+    def __call__(self) -> Optional[Callable]:
```

Comment:
Seems like `Optional[Callable]` is the wrong type hit. What does `self.callable()` return?

Reply:
Fixed